### PR TITLE
Miscellaneous release preparations

### PR DESCRIPTION
**Pull request by Lucki**
Some reordering and taking the latest changes into account (#76 #77 #82).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-American Truck Simulator (`-a`, `--ats`) or Euro Truck Simulator 2 (`-e`, `--ets2`)
-with steamcmd (`-u`, `--update`) and handles starting (`-s`, `--start`) the mod
-through Proton aka. Steam Play (`-p`, `--proton`) or Wine (`-w`, `--wine`).
-
-It needs a working Steam installation for starting through Proton or to update
-the game files. It will stop all running Steam processes while updating to
-prevent Steam asking for password and guard code at the next startup.
-When using standard Wine you should start the windows version of Steam first.
+It can install and update the windows version of American Truck Simulator and Euro Truck Simulator 2 with steamcmd and handles starting the multiplayer mod through Proton aka. Steam Play or Wine.
+It needs a working Steam installation for starting through Proton or when using standard Wine you should start the windows version of Steam first.
```
Comment (kakurasan) [inline README.md:6]: This PR is a good opportunity to replace "windows" with "Windows".

File: README.md [hunk 2]
```diff
@@ -52,183 +46,221 @@ Version|AppId
...
-### Using a different prefix location
+### Optional
 
-Note:
-* While the prefix for Wine will point directly to the prefix location,
-Proton uses a subfolder `pfx` for the actual prefix and points to the parent folder.
-* Your prefix must be 64bits, the mod is not 32bits-compatible.
+* [`wine`][repology:wine] as a possible replacement to Proton
+* [`git`][repology:git] to clone this repo and help developing
+* [`vdf`][python:vdf] to automatically detect the steam account with saved credentials
```
Comment (kakurasan) [inline README.md:119]: Sorry, I should have added the `pkg_resources` dependency here.
```markdown
* `pkg_resources` (part of [`setuptools`][python:setuptools]) to get version information from our Python package
```
Debian-based systems provide the `python3-pkg-resources` package.
This is not part of `python3-setuptools` package. `python3-setuptools` depends on `python3-pkg-resources`.

Comment (Lucki) [reply README.md:119]: This isn't even optional for installations as we don't have a version file there.

Comment (kakurasan) [reply README.md]: I think it's optional for Python package users too. `pkg_resources` is used only when `--version` option is specified.

We already have `setuptools` in `setup.cfg`:
```ini
[options.extras_require]
optional = setuptools; vdf
```

Comment (Lucki) [reply README.md]: Hm, it's kinda hard to give support without knowing what version they're running. But we can try, I've set it as optional.

File: README.md [hunk 3]
```diff
@@ -1,234 +1,262 @@
...
-the game afterwards.
+A simple launcher for TruckersMP to be used with Proton or Wine.
 
-It can install and update the windows version of
-American Truck Simulator (`-a`, `--ats`) or Euro Truck Simulator 2 (`-e`, `--ets2`)
-with steamcmd (`-u`, `--update`) and handles starting (`-s`, `--start`) the mod
-through Proton aka. Steam Play (`-p`, `--proton`) or Wine (`-w`, `--wine`).
+It can install and update the Windows version of American Truck Simulator and Euro Truck Simulator 2 with steamcmd and handles starting the multiplayer mod through Proton or Wine.
+A working native Steam installation is needed for starting through Proton - when using standard Wine you should have the Windows version of Steam already running.
+While updating the launcher will stop all running Steam processes to prevent Steam from asking for password and guard code at the next startup.
```
Comment (kakurasan) [inline README.md:7]: Proton and (Linux version of) steamcmd only support Linux systems.

Could you take other OSes into consideration? We need "on Linux" or something else for Linux-only stuff.

File: truckersmp_cli/__init__.py [hunk 4]
```diff
@@ -819,16 +819,14 @@ def check_args_errors():
...
-prevent Steam asking for password and guard code at the next startup.
-When using standard Wine you should start the windows version of Steam first.
+A simple launcher for TruckersMP to be used with Proton or Wine.
+It can install and update the Windows version of American Truck Simulator and
+Euro Truck Simulator 2 with steamcmd and handles starting the multiplayer mod
+through Proton or Wine. A working native Steam installation is needed for
+starting through Proton - when using standard Wine you should have the Windows
+version of Steam already running. While updating the launcher will stop all
+running Steam processes to prevent Steam from asking for password and guard code
+at the next startup.
```
Comment (kakurasan) [inline truckersmp_cli/__init__.py:829]: Same.

**Comment by kakurasan** [comment]
> Some `AppId`s are still untouched

* `./truckersmp_cli/__init__.py`:`    """Steam AppIds."""`
* `./truckersmp_cli/__init__.py`:`    # load Proton AppId info from "proton.json":`
* `./truckersmp_cli/__init__.py`:`    #     {"X.Y": AppId, ... , "default": "X.Y"}`
* `./.github/workflows/check_proton_json.yaml`:`      - name: Generate Proton AppId list from proton.json`
* `./.github/workflows/check_proton_json.yaml`:`                     epilog = "Proton AppId list:\n"`

Don't forget to combine the last commit and the additional changes into a single commit. Since we need multiple commits, I cannot do "Squash and merge" here.

**Comment by Lucki** [comment]
I've left them out because they aren't visible to the user.
I would squash them.

Is the wording ok?

**Pull request merged**